src/web/components/elements/TextBlockComponent.tsx

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
 import { css } from 'emotion';
-import { body } from '@guardian/src-foundations';
+import { body } from '@guardian/src-foundations/__typography';
 import { unescapeData } from '@root/src/lib/escapeData';
 // tslint:disable:react-no-dangerous-html
 
 const para = css`
     margin-bottom: 16px;
-    ${body({ level: 2 })};
+    ${body.medium()};
```

Comment:
It looks like the old levels 2, 3 and 5 have all become `medium` at various points in this file - just wanted to check that's right? (Probably just me missing context of what levels mean).

Reply:
Hi @tjmw 👋 

In the old API, the `level` maps to a particular font size, and doesn't really connote anything in and of itself. For instance, [`headline` level 3](https://github.com/guardian/source-components/blob/master/packages/foundations/src/typography.ts#L11), is a larger font size than `headline` level 1, but that's about all you can infer. To go further, [`body` level 1](https://github.com/guardian/source-components/blob/master/packages/foundations/src/typography.ts#L20) maps to a larger font size than [`textSans` level 1](https://github.com/guardian/source-components/blob/master/packages/foundations/src/typography.ts#L25).

With the new API, we have identified a medium font size for each font family. Again, the `medium` for headline might be larger than the medium for `body`. But by providing a medium font size, we're offering the developer a recommended starting point for the font family.

Hopefully this makes sense, but please let me know if I can do anything to make it clearer 😊

Reply:
Ah great, thanks for the explanation! That all makes sense :+1: